features/dd-sdk-android-session-replay/src/test/kotlin/com/datadog/android/sessionreplay/internal/recorder/TreeViewTraversalTest.kt

```diff
@@ -131,28 +138,55 @@ internal class TreeViewTraversalTest {
     @Test
     fun `M default to view mapper and continue W traverse { mapper for view type not provided }`(
```

Comment:
I think the description for this test may be wrong, since this test is traversing a single view